OsmAnd/src/net/osmand/plus/views/ContextMenuLayer.java

```diff
@@ -233,14 +233,19 @@ public void onDraw(Canvas canvas, RotatedTileBox box, DrawSettings nightMode) {
 			int x = (int) box.getPixXFromLatLon(latLon.getLatitude(), latLon.getLongitude());
 			int y = (int) box.getPixYFromLatLon(latLon.getLatitude(), latLon.getLongitude());
 			Bitmap pressedBitmapSmall = getBackground(pressedLatLonSmall.get(latLon), true);
-			Rect destRect = getIconDestinationRect(x, y, pressedBitmapSmall.getWidth(), pressedBitmapSmall.getHeight(), scale);
+			Rect destRect = getIconDestinationRect(
+					x, y, pressedBitmapSmall.getWidth(), pressedBitmapSmall.getHeight(), scale);
 			canvas.drawBitmap(pressedBitmapSmall, null, destRect, paint);
 		}
 		for (LatLon latLon : pressedLatLonFull.keySet()) {
 			int x = (int) box.getPixXFromLatLon(latLon.getLatitude(), latLon.getLongitude());
 			int y = (int) box.getPixYFromLatLon(latLon.getLatitude(), latLon.getLongitude());
-			Bitmap pressedBitmap = getBackground(pressedLatLonFull.get(latLon), false);
-			Rect destRect = getIconDestinationRect(x, y, pressedBitmap.getWidth(), pressedBitmap.getHeight(), scale);
+
+			BackgroundType background = pressedLatLonFull.get(latLon);
+			Bitmap pressedBitmap = getBackground(background, false);
+			int offsetY = BackgroundType.COMMENT.equals(background) ? pressedBitmap.getHeight() / 2 : 0;
```

Comment:
enum should be used with ==

Bad code, it should be used int offsetY = background.getOffset&(...)